docs/lua-modding/datatables.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 9
+---
+
+# DataTables
```

Comment:
add a note that they can only be read right now, and modification functionality is coming soon